examples/fluids/qfunctions/advection2d.h

```diff
@@ -36,6 +36,8 @@ struct AdvectionContext_ {
   int stabilization; // See StabilizationType: 0=none, 1=SU, 2=SUPG
 };
 
+CEED_QFUNCTION_HELPER CeedScalar Square(CeedScalar x) { return x*x; }
```

Comment:
Probably could put this in a common header shared between the QFunctions with an include via rpath

Reply:
Yeah, I didn't want to create that coupling just yet, but I envision a common header with utilities for scalars and 3x3 matrices.